spec/user_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+describe User do
+
+  context 'Checking DataMapper can create and delete entries' do
```

Comment:
I know this kind of thing was in the BookmarkManager tutorial but it's not necessarily best practice. Check out the following for improving upon this: http://matchers.shoulda.io/